Makefile

```diff
@@ -5,7 +5,7 @@ PREVIOUS_RELEASE=1.2.0
 
 R2R=radare2-regressions
 R2R_URL=$(shell doc/repo REGRESSIONS)
-R2BINS=$(shell cd binr ; echo r*2 r2agent r2pm)
+R2BINS=$(shell cd binr ; echo r*2 r2agent r2pm r2-indent)
```

Comment:
we should split R2BINS and R2SCRIPTS, because shellscripts cant be stripped. and right now, r2-indent only works in symlink mode